src/main/java/de/timeline/TimelineController.java

```diff
@@ -30,8 +30,10 @@ public void createNewEvent(@RequestBody Event event) {
 		timelineService.createEvent(event);
 	}
 
-	@DeleteMapping("/events")
-	public ResponseEntity deleteOldEvent(@PathVariable UUID uuid) {
+	@DeleteMapping("/events/{uuid}")
+	// hier fehlte die uuid noch als Platzhalter im Mapping, das war sicher im
+	// Video so gezeigt!
+	public ResponseEntity<?> deleteOldEvent(@PathVariable UUID uuid) {
```

Comment:
Der Rückgabewert vom Typ ResponseEntity bestimmt den Response-Body. Wenn da stehen würde ResponseEntity<Event> würde ein Event in den Response-Body geschrieben. Für den Fall ResponseEntity<List<Event>> mehrere Events. ResponseEntity ist also ein Generic-Typ, der innerhalb der spitzen Klammern anzeigt, welche Nutzlast er tragen kann. Der Sonderfall ? zeigt an, dass entweder alles oder nichts rein kann. In Deinem Fall ist es nichts, weil wir ja außer einem Status-Code nichts zurückgeben, auch keinen Response-Body.